src/Common/com/bioxx/tfc/ClientProxy.java

```diff
@@ -661,8 +520,7 @@ public boolean getGraphicsLevel()
 	@Override
 	public void hideNEIItems()
 	{
-		String mod = "NotEnoughItems";
-		if (Loader.isModLoaded(mod) && TFCCrafting.enableNEIHiding)
+		if (Loader.isModLoaded("NotEnoughItems") && !System.getProperties().containsKey("tfc.disableNEIhiding"))

```

Comment:
This needs to be changed back to use the config option and not the system property.